Dockerfile

```diff
@@ -32,6 +32,7 @@ COPY webpack webpack
 ENV NODE_ENV "production"
 ARG CI_COMMIT_SHA
 ENV CI_COMMIT_SHA ${CI_COMMIT_SHA}
+RUN echo "{ \"version\": \"${CI_COMMIT_SHA}\", \"date\":\"$(date)\" }" > app/version.json
```

Comment:
Is it confusing to have this information injected into the build at two different points (here and in the webpack build)? I think either is fine but I would prefer to have only one.

Reply:
I vote for this method - as config can be changed :) ... as its meant to be!

Reply:
I think tam is referring to the fact that we also inject this into the html as a comment, not that it's taken off config.

The sha is added to config and then baked into the build as a comment, or here instead if you wish.

Reply:
I don't understand... where is it injected into the html? I certainly cannot see it on any of the sites.

Reply:
Injecting into the HTML is not enough. Product and other stakeholders should not be expected to inspect source- this information needs to be readily available and immeadiately visible.